apps/koillection/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '3'
+
+services:
+    koillection:
+        image: koillection/koillection:1.4.11
+        container_name: koillection
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:80
+        depends_on:
+            - db
+        volumes:
+            - ${APP_DATA_DIR}/data/volumes/koillection/uploads:/uploads
```

Comment:
```suggestion
            - ${APP_DATA_DIR}/data/uploads:/uploads
```

Reply:
should postgresql change path to :
            - ${APP_DATA_DIR}/data/postgresql:/var/lib/postgresql/data 
or keep 
            - ${APP_DATA_DIR}/data/volumes/postgresql:/var/lib/postgresql/data ?